packages/stack/src/views/Stack/CardStack.tsx

```diff
@@ -299,7 +300,7 @@ export default class CardStack extends React.Component<Props, State> {
           props.insets,
           state.descriptors,
           layout,
-          {}
+          state.headerHeights
```

Comment:
I'm not sure why this was previously an empty object. It had the effect of reseting `headerHeights` to be based on `getDefaultHeaderHeight` every time the scene container calls `onLayout`, which led to jittery behavior. Would be helpful to know the background for this decision, curious if there's an edge case I missed in testing

Reply:
Looks like a mistake